File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/3040.cpp
#include <iostream>

using namespace std;

int main(){
    int n, a, b, c;
    cin >> n;
    for(int i=0;i<n;i++){
        cin >> a >> b >> c;
        if(a>=200 && a<=300 && b>=50 && c>=150){
            cout << "Sim" << endl;
        } else{
            cout << "Nao" << endl;
        }
    }

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/3040.cs
using System;

class URI
{
    static void Main() {
        int i, n, a, b, c;
        n = int.Parse(System.Console.ReadLine().Trim());
        for(i=0;i<n;i++){
            string nums = System.Console.ReadLine().Trim();
            string[] ns = nums.Split(' ');
            a = int.Parse(ns[0]);
            b = int.Parse(ns[1]);
            c = int.Parse(ns[2]);
            if(a>=200 && a<=300 && b>=50 && c>=150){
                Console.Write("Sim\n");
            } else{
                Console.Write("Nao\n");
            }
        }
    }
}